openwisp-utils-qa-checks

```diff
@@ -58,9 +59,16 @@ runcheckmigrations() {
     FAILURE=1
     return
   fi
-  checkmigrations --migration-path "$MIGRATION_PATH" --migrations-to-ignore "$MIGRATIONS_TO_IGNORE" \
-    && echo "SUCCESS: Migration name check successful!" \
-    || { echoerr "ERROR: Migration name check failed!"; FAILURE=1; }
+  migrations_count="${#MIGRATION_PATH[@]}"
+  for i in "${!MIGRATION_PATH[@]}"
+  do
+    path="${MIGRATION_PATH[$i]}"
+    ignore="`[ ${MIGRATIONS_TO_IGNORE[$i]} ] && echo \"${MIGRATIONS_TO_IGNORE[$i]}\" || echo 0`"
```

Comment:
Can you please maintain the output like the other tests ("SUCCESS", "ERROR"), like:
"SUCCESS: Migration name check on $path with migrations-to-ignore: $ignore"
"ERROR: Migration name check on $path with migrations-to-ignore: $ignore"
and remove the "Running.." line.

Reply:
@atb00ker I did add something though before that like
```
Running migration name check...
[1/2] Checking on migration-path: $path with migrations-to-ignore: $ignore
<Traceback error>
ERROR: Migration name check failed!
[2/2] Checking on migration-path: <path> with migrations-to-ignore: <ignore>
<Traceback error>
ERROR: Migration name check failed!
```
It has a nice friendly section first then the traceback logs error instead of the one you propose that'll be like:
```
<Traceback error>
ERROR: Migration name check on $path with migrations-to-ignore: $ignore
<Traceback error>
ERROR: Migration name check on $path with migrations-to-ignore: $ignore
```
That write the traceback logs first then the error instead.
But if you really want to make it like the one you propose then okay I will change it, I just to make sure :smile:

Reply:
Actually, I think 
```
<Traceback error>
ERROR: Migration name check on $path with migrations-to-ignore: $ignore
```
sounds good, even python does it that way and in most cases only. If it doesn't work out, we can always get back to this approach, what do you think? :smile:

Reply:
:smile: Fair enough. Okay, I will change it